### PR TITLE
Add HUK24

### PR DESCRIPTION
**Pull request by juliushacker**

### CODE REVIEW

File: entries/h/huk24.de.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "HUK24": {
+    "domain": "huk24.de",
+    "contact": {
+      "email": "info@huk24.de",
+      "language": "de"
+    },
```
Comment (hkamran80) [inline entries/h/huk24.de.json]:
```suggestion
```

The `contact` object should only be present if the site does not support 2FA

File: img/h/huk24.de.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="193" height="193" version="1.1" viewBox="155 -65 193 193" xmlns="http://www.w3.org/2000/svg">
+<path d="m241.6 33.81a25.24 25.24 0 0 0 5.4-15.4 18.38 18.38 0 0 0-5-13.08q-5-5.33-15.89-5.33a26.43 26.43 0 0 0-11.35 2.41 19.5 19.5 0 0 0-8.39 7.59q-3.23 5.16-3.49 13.16h18.5q0-4.12 0.73-5.63a2.52 2.52 0 0 1 2.45-1.53c2 0 3 1.44 3 4.3q0 5.25-6.59 11.88a170.24 170.24 0 0 1-17.85 15.14v14.37h44.48v-15.4h-19.7a51.07 51.07 0 0 0 13.7-12.48zm61.4 2.3h-6.1v-35.42h-20.26l-25.25 36v15.7h26.52v9.32h19v-9.32h6.09zm-23.56 0h-9.15l9.15-14z" fill="#f49b08"/>
+<path d="m325.38 0.53h-24.29l23.09 30.32-22.22 30.86h24.44l20.43-30.86zm-124.9 61.18-23.09-30.32 22.21-30.86h-24.44l-19.01 30.795 20.04 30.385z" fill="#35444c"/>
+</svg>
```
Comment (hkamran80) [inline img/h/huk24.de.svg:5]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="155 -65 190.7 61.7"><path fill="#f49b08" d="M240.45-31.188a25.24 25.24 0 0 0 5.4-15.4 18.38 18.38 0 0 0-5-13.08q-5-5.33-15.89-5.33a26.43 26.43 0 0 0-11.35 2.41 19.5 19.5 0 0 0-8.39 7.59q-3.23 5.16-3.49 13.16h18.5q0-4.12.73-5.63a2.52 2.52 0 0 1 2.45-1.53c2 0 3 1.44 3 4.3q0 5.25-6.59 11.88a170.24 170.24 0 0 1-17.85 15.14v14.37h44.48v-15.4h-19.7a51.07 51.07 0 0 0 13.7-12.48zm61.4 2.3h-6.1v-35.42h-20.26l-25.25 36v15.7h26.52v9.32h19v-9.32h6.09zm-23.56 0h-9.15l9.15-14z"/><path fill="#35444c" d="M324.23-64.468h-24.29l23.09 30.32-22.22 30.86h24.44l20.43-30.86zm-124.9 61.18-23.09-30.32 22.21-30.86h-24.44L155-33.673l20.04 30.385z"/></svg>
```

Here's a minified SVG

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/h/huk24.de.json (hunk 1 above)
Comment (Carlgo11) [inline entries/h/huk24.de.json:7]: As the site already supports 2FA there's no reason for contact information to be stored :wink: 
```suggeston
```

File: img/h/huk24.de.svg (hunk 2 above)
Comment (Carlgo11) [inline img/h/huk24.de.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 190.6 61.7"><path fill="#f49b08" d="M85.4 33.8a25.2 25.2 0 0 0 5.4-15.4 18.4 18.4 0 0 0-5-13Q80.8 0 69.9 0a26.4 26.4 0 0 0-11.3 2.4 19.5 19.5 0 0 0-8.4 7.6q-3.3 5.2-3.5 13.2h18.5q0-4.2.7-5.7a2.5 2.5 0 0 1 2.5-1.5c2 0 3 1.4 3 4.3q0 5.3-6.6 11.9a170.2 170.2 0 0 1-18 15.1v14.4h44.5V46.3H71.6a51 51 0 0 0 13.7-12.5zm61.4 2.3h-6.1V.7h-20.3l-25.2 36v15.7h26.5v9.3h19v-9.3h6.1zm-23.6 0h-9.1l9.1-14z"/><path fill="#35444c" d="M169.2.5h-24.4l23 30.5-22 30.7h24.4l20.4-30.9zm-125 61.2-23-30.3L43.4.5H19L0 31.3l20 30.4z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**